src/node/transaction.cpp

```diff
@@ -36,8 +36,17 @@ TransactionError BroadcastTransaction(NodeContext& node, const CTransactionRef t
         if (!existingCoin.IsSpent()) return TransactionError::ALREADY_IN_CHAIN;
     }
     if (!node.mempool->exists(hashTx)) {
-        // Transaction is not already in the mempool. Submit it.
+        // Transaction is not already in the mempool.
         TxValidationState state;
+        CAmount fee;
+        bool test_accepted = AcceptToMemoryPool(*node.mempool, state, std::move(tx),
```

Comment:
IMO you don't need the intermediate return value and can just move the call into the if() but if you want to keep it:
```suggestion
        const bool test_accepted = AcceptToMemoryPool(*node.mempool, state, std::move(tx),
```

Reply:
moved it into the if(), thanks!